docs/AnalyticObserving.md

```diff
@@ -0,0 +1,66 @@
+# Observing Analytics
+
+An [`AnalyticsListener`](https://appcues.github.io/appcues-android-sdk/appcues/com.appcues/-analytics-listener/index.html) can be registered to allow for the host application to see all of the analytics that are being tracked by the Appcues Android SDK.
+
+## Registering the AnalyticsListener
+
+One of the types in your application can implement the [`AnalyticsListener`](https://appcues.github.io/appcues-android-sdk/appcues/com.appcues/-analytics-listener/index.html) interface. In the usage example below, this is implemented on the main `Application` subclass. This interface defines a single function [`trackedAnalytic(type, value, properties, isInternal)`](https://appcues.github.io/appcues-android-sdk/appcues/com.appcues/-analytics-listener/tracked-analytic.html), which provides access to the analytics tracking being done by the SDK.
+
+```kotlin
+class ExampleApplication : Application(), AnalyticsListener {
+    override fun trackedAnalytic(type: AnalyticType, value: String?, properties: Map<String, Any>?, isInternal: Boolean) {
+        // process and use the analytics tracking information
+    }
+}
+```
+
+Set the listener for the Appcues Android SDK to use. In the usage example below, this is done in the ExampleApplication `onCreate()`. The listener can be provided during SDK initialization as a configuration property, as shown below, or set later after the SDK initialization completes.
+
+```kotlin
+override fun onCreate() {
+    super.onCreate()
+
+    appcues = Appcues(this, "APPCUES_ACCOUNT_ID", "APPCUES_APPLICATION_ID") {
```

Comment:
I think maybe this could be simplified by changing the whole interface on ExampleApplication to a private variable like:

kotlin
```
private val analyticsListener = object : AnalyticsListener { ... }
```

then onCreate:

kotlin
```
appcues = Appcues(this, "APPCUES_ACCOUNT_ID", "APPCUES_APPLICATION_ID") {
    analyticsListener = this.analysitcsListener
}
```

or it could be all in same line

kotlin
```
appcues = Appcues(this, "APPCUES_ACCOUNT_ID", "APPCUES_APPLICATION_ID") {
    analyticsListener = object : AnalyticsListener { ... }
}
```

Reply:
yeah basically just something has to implement that interface - this seems like a nice simplification for docs purposes though, will try it out! thanks